packages/cli/src/lib/flags/snapshotCommonFlags.ts

```diff
@@ -21,3 +21,14 @@ export const sync = () => ({
       'Apply synchronization when there is a 100% match between organization and snapshot resources.',
   }),
 });
+
+export const previewLevel = () => ({
+  previewLevel: flags.enum({
+    char: 'p',
+    description:
+      'The verbosity of the preview. The `light` preview is faster to generate but only contains a limited amount of information, as opposed to the `detailed` preview which takes more time to generate, but returns a diff representation of all the changes to apply.',
```

Comment:
```suggestion
      'The verbosity of the preview. The `light` preview is faster to generate but only contains a limited amount of information, as opposed to the `detailed` preview that takes more time to generate, but returns a diff representation of all the changes to apply.',
```

Reply:
(just a grammar thing. 'which' needs to be preceded by a comma, whereas 'that' doesn't)

Reply:
Thanks! I'll remember that.